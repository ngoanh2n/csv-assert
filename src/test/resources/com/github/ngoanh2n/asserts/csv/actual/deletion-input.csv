id,email,firstname,lastname,age,note
1,tester01@csvassert.vn,"Tester 01",Nguyen,21,