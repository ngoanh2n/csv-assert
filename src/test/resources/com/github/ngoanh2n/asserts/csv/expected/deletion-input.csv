id,email,firstname,lastname,age,note
,tester01@csvassert.vn,"Tester 01",Nguyen,,
,tester02@csvassert.vn,"Tester 02",Nguyen,,